### PR TITLE
Feat: Introduction section of Home Page

### LINKED ISSUE #109: Coding : Develop Introduction section of Home Page
### Is your feature request related to a problem? Please describe.
As a developer,
I need the following section to be developed so that I can add is as a Home component [here](https://github.com/anitab-org/anitab-org.github.io/tree/develop/src/Components)

### Mocks
Develop the section in red box only.

Screenshot (27)_LI

Font Size for heading EVENTS: Header Medium(32pt) weight- Medium
Font Size for the description below: body 18pt weight- Light
Font Style- Avenir
Font Color (heading): #103B81
Font Color (Body): #103B81

[Design link for development](https://xd.adobe.com/view/09a9fdf0-a193-4b3f-47c2-216055c7ca42-7f98/)

### Update [Required]
- [ ] Should follow all the margin guidelines mentioned above and should be responsive

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
6-8 hours

### PR DESCRIPTION
**Pull request by tichnas**
### Description

Introduction section of the Home Page is developed with links to Medium, Github and Zulip.
The code takes data dynamically from a file `src/content/home.js`, but the real content is not yet added (#108)

Fixes #109

### Type of Change:

- Code
- User Interface
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?

recording(3)-min

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/content/home.js [hunk 1]
```diff
@@ -1,4 +1,14 @@
 const home = {
+  introduction: {
+    title: 'AnitaB.org Open Source',
+    image: {
+      source: require('./../assets/landing/landing.png'),
+    },
+    content: {
+      par:
+        'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quis omnis minus, quisquam ab architecto dolor hic dolores, ducimus explicabo, voluptates fugit dolorum itaque enim nam. Enim nulla, vitae recusandae est pariatur consequuntur aperiam molestiae, minus vero, fugiat ipsam facilis cupiditate alias dolorem mollitia officiis. Excepturi, blanditiis?',
```
Comment (devkapilbansal) [inline src/content/home.js:9]: Why random text is used here? Is it intentional?

Comment (tichnas) [reply src/content/home.js:9]: Thanks for pointing this out. The real content is not there yet, so I used random text. I've also updated the description to inform this.

**Review by annabauza — CHANGES_REQUESTED** [review]
Thank you for that, however no Lorem ipsum will be merged. Also it feels like sometimes text is touching the image. Can you please make a testing version following [those steps](https://github.com/anitab-org/anitab-org.github.io/wiki/Prepare-testing-GitHub-Pages-for-your-PR) and ping our testing team. Thank you.

**Comment by tichnas** [comment]
@annabauza You're right about the text touching the images. It's a bug in an already build component (`ImageContent`) and I've created an issue (#167) for this which is solved in PR #170

**Comment by Vishal-raj-1** [comment]
@tichnas any updates ?

File: src/content/home.js (hunk 1 above)
Comment (annabauza) [reply src/content/home.js:9]: no `Lorem ipsum` will be merged. Rise issue and wait for content or write content by your own. Thanks!

**Review by annabauza — CHANGES_REQUESTED** [review]
Replace Lorem ipsum with real paragraph text.

**Comment by annabauza** [comment]
@tichnas I close this PR due to lack of respond.

**Pull request closed without merging**